adapter/persistent/mu-persistent.cabal

```diff
@@ -0,0 +1,23 @@
+name:                mu-persistent
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/higherkindness/mu-haskell/persistent#readme
+license:             Apache-2.0
+license-file:        LICENSE
+author:              Flavio Corpa
+maintainer:          flavio.corpa@47deg.com
+copyright:           Copyright © 2019-2020 47 Degrees. <http://47deg.com>
+category:            Network
+build-type:          Simple
+cabal-version:       >=1.10
+extra-source-files:  README.md
+
+library
+  exposed-modules:     Mu.Adapter.Persistent
+  hs-source-dirs:      src
+  default-language:    Haskell2010
+  build-depends:       base >= 4.7 && < 5
+                     , mu-schema
+                     , persistent
+                     , persistent-sqlite
```

Comment:
We should not require a specific database library here. I think this can be removed without any problem (other than adjusting imports).